sag_ji4.par

```diff
@@ -1,306 +1,306 @@
+-13.420170678011700	,'~!(	5589/5632	,'o	-27
+-12.998594635770500	'~!(	135/136	'o	-26
+-12.436493246115600	)~!	143/144		-25
+-12.014917203874400	.)!(	2816/2835	.i	-24
+-11.452815814219400	`.)!(	2083725/2097152	`.i	-23
+-11.031239771978200	,,)!(	106496/107163	,,i	-22
+-10.469138382323300	,)!(	2187/2200	,i	-21
+-10.047562340082100	)!(	891/896	i	-20
+-9.485460950427140	`)!(	131072/131769	`i	-19
+-9.063884908185930	~!	2176/2187	s	-18
+-8.501783518530990	`~!	1024/1029	`s	-17
+-8.080207476289780	.!(	224/225	.n	-16
+-7.518106086634840	`.!(	19600/19683	`.n	-15
+-7.096530044393640	'~!	255/256	s	-14
+-6.534428654738690	,!(	10935/10976	,n	-13
+-6.112852612497490	!(	5103/5120	n	-12
+-5.550751222842550	`!(	22528/22599	`n	-11
+-5.129175180601340	``!(	351/352	``n	-10
+-4.567073790946400	,,)!	384/385	,,;	-9
+-4.145497748705190	,)!	440/441	,;	-8
+-3.723921706463990	)!	512/513	;	-7
+-3.161820316809050	`)!	1120000/1121931	`;	-6
+-2.740244274567840	,.!	728/729	,.|	-5
+-2.178142884912900	.!	32768/32805	.|	-4
+-1.756566842671690	')!	1215/1216	';	-3
+-1.194465453016750	,,!	2079/2080	,,!	-2
+-0.772889410775545	,!	4095/4096	,!	-1
+-0.210788021120603
+0.210788021120603	`|	4096/4095	`|	1
+0.772889410775545	``|	2080/2079	``|	2
+1.194465453016750	.)|	1216/1215	.r	3
+1.756566842671690	'|	32805/32768	'|	4
+2.178142884912900	`'|	729/728	`'|	5
+2.740244274567840	,)|	1121931/1120000	,r	6
+3.161820316809050	)|	513/512	r	7
+3.723921706463990	`)|	441/440	`r	8
+4.145497748705190	``)|	385/384	``r	9
+4.567073790946400	,,|(	352/351	,,u	10
+5.129175180601340	,|(	22599/22528	,u	11
+5.550751222842550	|(	5120/5103	u	12
+6.112852612497490	`|(	10976/10935	`u	13
+6.534428654738690	.~|	256/255	.$	14
+7.096530044393640	,'|(	19683/19600	,'u	15
+7.518106086634840	'|(	225/224	'u	16
+8.080207476289780	,~|	1029/1024	,$	17
+8.501783518530990	~|	2187/2176	$	18
+9.063884908185930	,)|(	131769/131072	,*	19
+9.485460950427140	)|(	896/891	*	20
+10.047562340082100	`)|(	2200/2187	`*	21
+10.469138382323300	``)|(	107163/106496	``*	22
+11.031239771978200	,')|(	2097152/2083725	,'*	23
+11.452815814219400	')|(	2835/2816	'*	24
+12.014917203874400	)~|	144/143		25
+12.436493246115600	.~|(	136/135	.e	26
+12.998594635770500	`.~|(	5632/5589	`.e	27
+13.420170678011700	,,~|(	126/125	,,e	28
+13.841746720252900	,~|(	245/243	,e	29
+14.403848109907900	~|(	4131/4096	e	30
+14.825424152149100	`~|(	572/567	`e	31
+15.387525541804000	,,|~	413343/409600	,,~	32
+15.809101584045200	,|~	33075/32768	,~	33
+16.371202973700200	|~	736/729	~	34
+16.792779015941400	`|~	512/507	`~	35
+17.354880405596300	~~|	99/98		36
+17.776456447837500	`~~|	8019/7936		37
+18.338557837492500	``~~|	139264/137781		38
+18.760133879733700	,./|	91/90	,./	39
+19.322235269388600	./|	2048/2025	./	40
+19.743811311629800	)|~	19683/19456		41
+20.305912701284800	,,/|	256/253	,,/	42
+20.727488743526000	,/|	66339/65536	,/	43
+21.289590133180900	/|	81/80	/	44
+21.711166175422100	`/|	875/864	`/	45
+22.132742217663300	``/|	3200/3159	``/	46
+22.694843607318300	.)/|	76/75		47

```

Comment:
interestingly, this was fine, though there was a mistake in the downwards section